xfce-extra/xfce4-cpugraph-plugin/files/1.1.0-panel-4.15.patch

```diff
@@ -0,0 +1,12 @@
+diff -Naur xfce4-cpugraph-plugin-1.1.0/panel-plugin/cpu.h xfce4-cpugraph-plugin-1.1.0-patched/panel-plugin/cpu.h
```

Comment:
Please use a git patch with correct metadata. You can grab it from: https://gitlab.xfce.org/panel-plugins/xfce4-cpufreq-plugin/-/commit/30f6de22897798cb1d8b6a18d506e9e331a97b10.patch

Also start the filename with package name, i.e. xfce4-....patch

Reply:
Just a note: This pull request is for cpugraph plugin, not cpufreq plugin.

Reply:
Yeah, sorry, gave you the wrong link.